### ISSUE TITLE
Docker build fails on node:14-alpine due to Next.js 12.0.2 issue - Upgrade to Next.js 12.0.3 is required

### ISSUE THREAD

**Issue by cb5890** [calendso/calendso#1148]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

It's currently not possible to build calendos's docker image (see [1]) due to an issue in next.js 12.0.2. The build fails with
```
#14 0.432 $ next build
#14 1.287 warn  EMAIL_FROM environment variable is not set, this may indicate mailing is currently disabled. Please refer to the .env.example file.
#14 1.314 Attention: Next.js now collects completely anonymous telemetry regarding usage.
#14 1.315 This information is used to shape Next.js' roadmap and prioritize features.
#14 1.315 You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
#14 1.315 https://nextjs.org/telemetry
#14 1.315
#14 1.371 info  - Skipping validation of types...
#14 1.605 info  - Creating an optimized production build...
#14 3.032 Error: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /app/node_modules/@next/swc-linux-x64-gnu/next-swc.linux-x64-gnu.node)
#14 3.032     at Object.Module._extensions..node (internal/modules/cjs/loader.js:1144:18)
#14 3.032     at Module.load (internal/modules/cjs/loader.js:950:32)
#14 3.032     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
#14 3.032     at Module.require (internal/modules/cjs/loader.js:974:19)
#14 3.032     at require (internal/modules/cjs/helpers.js:93:18)
#14 3.032     at Object.<anonymous> (/app/node_modules/next/dist/build/swc/index.js:29:20)
#14 3.032     at Module._compile (internal/modules/cjs/loader.js:1085:14)
#14 3.032     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
#14 3.032     at Module.load (internal/modules/cjs/loader.js:950:32)
#14 3.032     at Function.Module._load (internal/modules/cjs/loader.js:790:12) {
#14 3.032   code: 'ERR_DLOPEN_FAILED'
#14 3.032 }
#14 3.032 error - Failed to load SWC binary, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
```

The issue has been acknowledged in Next.js 12.0.2 and was fixed in 12.0.3 (see [2]).

From my point of view, fixing the docker image build just requires bumping the Next.js version up to 12.0.3 in the `package.json` (see [3]). Then, everything should be working again as intended. 

### Steps to Reproduce

1. Build the docker image using the instructions provided in [1].

[1] https://github.com/calendso/docker
[2] https://github.com/vercel/next.js/issues/30713
[3] https://github.com/calendso/calendso/blob/a404ca847c731f594087b2e2cc5834143c216adb/package.json#L70

**Comment by Kovah** [comment]
The upgrade to Next.js 12.0.3 is not really necessary. I already opened a pull request with a fix to the Dockerfile.

https://github.com/calendso/docker/pull/41

**Comment by cb5890** [comment]
Yes, well, both options seem to work. But you can leave build and runtime image the same.... and it often doesn't hurt to migrate to a newer version - especially when the release lists quite some fixes ;-)

The binaries that are required for the Next.js compilation are apparently downloaded to the build container for 12.0.3. However, I don't mind either way :)

**Comment by zephleggett** [comment]
I resolved in my setup by using Next 12.0.3
